sktime/forecasting/timesfm_forecaster.py

```diff
@@ -208,6 +211,28 @@ def __init__(
         os.environ["JAX_PLATFORM_NAME"] = backend
         os.environ["JAX_PLATFORMS"] = backend
 
+        # Deprecation warning for the repo_id default change
+        warn(
+            f"{self.__class__.__name__}: "
+            f"The 'timesfm' implementation in sktime fork will "
+            "switch to a pytorch-based from jax-based in release 0.33.3. "
+            "As a result, the default value for the 'repo_id' "
+            f"parameter will change from '{repo_id}' "
+            "to 'google/timesfm-1.0-200m-pytorch'.",
+            DeprecationWarning,
+        )
+
+        # Deprecation warning for the changing dependencies
```

Comment:
todos should be added, e.g., `todo version: do this`

Reply:
since there are going to be significant changes with the estimator, I thought I could just do this, does it work?
```py
# TODO 0.35.0: remove this class with the
# pytorch based class in https://github.com/sktime/sktime/pull/7155
class TimesFMForecaster(_BaseGlobalForecaster):
    ...
```